README.md

```diff
@@ -132,8 +132,13 @@ Support this project with a :star:
     # Default: "false"
     fail-if-changed: ''
 
-    # Message to display when files have changed and the 
-    # `fail-if-changed` input is set to `true`. 
+    # Indicates whether to fail if no files have changed.
```

Comment:
The changes in this file can be reverted since it’s automatically generated

Reply:
Missed that, reverted it